wfcGen.py

```diff
@@ -93,7 +89,7 @@ def __init__(self, gp):
         self.entropy = []
         self.checkCollapsedNeighbours = True
 
-    def collapse(self):
+    def collapse(self, cells):

```

Comment:
I think i will make Grid class to take cells as a parameter when created
class Cell requiers whole cells dictionary to collapse, but I will fix getEntropy function, so it will not use cells dict

Reply:
Alright. Didn't get all of what you said but let's try to do that